README.md

```diff
@@ -226,6 +226,58 @@ You will see this result in the response to your `curl` call to the predict endp
 
 Now you've seen how easy it can be to serve a deep learning model with TorchServe! [Would you like to know more?](docs/server.md)
 
+### Get metrics
+
+The default metrics endpoint returns Prometheus formatted metrics. The endpoint can be set to a different port via config parameter `metrics_address`. You can query metrics using curl requests as below
+
+```
+curl http://127.0.0.1:8082/metrics
+curl "http://127.0.0.1:8082/metrics?name[]=ts_inference_requests_total&name[]=ts_queue_latency_microseconds" --globoff
+```
+
+```
+# HELP ts_inference_latency_microseconds Cumulative inference duration in microseconds
```

Comment:
Split this into multiple comments based on the metric type to make it more readable. Also add the respective curl commands for each type as an example.

Reply:
Done